doc/Example-Gradle-Project/build.gradle

```diff
@@ -11,6 +11,19 @@ apply plugin: 'application'
 
 mainClassName = 'org.sosy_lab.java_smt_example.SolverOverviewTable'
 
+// Globally define versions used for our dependencies later
+def javaSMTVersion = '3.11.0'
+def javaSMTYices2Version = '3.10.0'
+def princessVersion = '2021-11-15'
+def smtInterpolVersion = '2.5-916-ga5843d8b'
+def boolectorVersion = '3.2.2-gcc3a7091'
+def z3Version = '4.8.10'
+def yices2Version = '2.6.2-396-g194350c1'
+def cvc4Version = '1.8-prerelease-2020-06-24-g7825d8f28'
+def mathsat5Version = '5.6.6'
+def optiMathsat5Version = '1.7.1-sosy0'
+def junit4Version = '4.13'
```

Comment:
You could put (a dense representation of) the solver dependencies in a map. The syntax is a bit lighter to the eye. If you add a little more information than just the versions, you even iterate over the map in the `dependencies` block.

```groovy
def solverVersions = [
'yices2': '3.10.0',
'princess': '2021-11-15',
// ...
];
```

Reply:
I like your map better ;D as you can see in the latest commit. Thank you!

You can even use it like this: solverVersions.yices2